main/index.js

```diff
@@ -63,6 +66,12 @@ const ctx = {
     ipcMain.emit(ipcMainEvents.ACTIVITY_LOGGED, activityLog.getAllEntries())
   },
 
+  getTotalJobCount: () => jobCounter.getTotalJobCount(),
```

Comment:
If you agree with the above comment, `getTotalJobsCompleted` would fit here nicely for example